### PR TITLE
update for Chinese Localization

### PR DESCRIPTION
**Pull request by sdbds**
translation some string to Chinese

### CODE REVIEW

File: Hearthstone Deck Tracker/Properties/Strings.zh-CN.resx [hunk 1]
```diff
@@ -118,6 +118,1120 @@
...
+    <value>另存为</value>
+  </data>
+  <data name="MainWindow_DeckBuilder_Button_Cancel" xml:space="preserve">
+    <value>取消</value>
+  </data>
+  <data name="MainWindow_DeckBuilder_Button_Filter" xml:space="preserve">
+    <value>筛选</value>
+  </data>
+  <data name="MainWindow_DeckBuilder_Filter_Cost" xml:space="preserve">
+    <value>消耗</value>
```
Comment (jindaxia) [inline Hearthstone Deck Tracker/Properties/Strings.zh-CN.resx]: I think 费用 would be better

Comment (jindaxia) [inline Hearthstone Deck Tracker/Properties/Strings.zh-CN.resx]: I think 费用 would be better

**Comment by azeier** [comment]
I moved the Strings files to a separate repo: https://github.com/HearthSim/HDT-Localization

Could you create a new PR over there?

**Comment by sdbds** [comment]
OK，got it.

**Pull request closed without merging**